docs/v2-system-addon/snippets.md

```diff
@@ -36,10 +36,6 @@ a Promise that resolves when the blockList has been written to indexedDB.
 `.showFirefoxAccounts()`: (func) A function that opens the Firefox Accounts signup page
 (specifically `about:accounts?action=signup&entrypoint=snippets`)
 
-`.disableOnboarding()`: (func) A function that disables onboarding for future new tabs (until Firefox restarts).
```

Comment:
I would leave this in and note that it's deprecated as of 65 since onboarding tour is no longer around. Removing it completely complicates the code for snippets running on older versions